spec/controllers/api/scores_controller_spec.rb

```diff
@@ -104,4 +104,26 @@
       expect(Score.count).to eq score_count
     end
   end
+
+  describe 'GET user_scores' do
+    it 'should return the scores related to the user' do
+      @user = create(:user, name: 'User', email: 'user7@email.com', password: 'userpass')
+
+      sign_in(@user, scope: :user)
+
+      @score1 = create(:score, user: @user, total_score: 79, played_at: '2021-05-20')
+      @score2 = create(:score, user: @user, total_score: 59, played_at: '2021-05-25')
+
+      get api_golfers_path(user_id: @user.id)
```

Comment:
Please write another test for the case when you request an user's scores, logged in as another user.